docs/docs/quickstart/README.md

```diff
@@ -211,12 +211,12 @@ curl -d '{
 
 The event will instantly appear in the event list. Select it to inspect the workflow execution.
 
-![image-20220329011500931](./v2/README/image-20220329011500931.png)
+![image-20220329011500931](https://res.cloudinary.com/pipedreamin/image/upload/v1675691952/fe9eizdn8fwmwmgegb2m.png)
 
 Finally, you can return to Google Sheets to validate that the new data was automatically inserted.
 
 ![image-20220329011543065](./v2/README/image-20220329011543065.png) 
 
 ## Next Steps
 
-Congratulations! You completed the quickstart and should now understand the basic patterns for workflow development. Next, try creating your own workflows and check out the docs to learn more!
+Congratulations! You completed the quickstart and should now understand the basic patterns for workflow development. Next, try creating your own workflows and check out the [docs](/) to learn more!
```

Comment:
```suggestion
Congratulations! You completed the quickstart and should now understand the basic patterns for workflow development. Next, try creating your own workflows and check out the [documentation](/) to learn more!
```

Reply:
Changed 'docs' to 'documentation' to be more descriptive (Google's Developer docs style guide)